lib/goals/build.js

```diff
@@ -37,8 +37,5 @@ module.exports = async function build (config) {
   await imageStreamConfigurator.createOrUpdateImageStream(config);
 
   // push to the cluster
-  await binaryBuild(config, `${config.projectLocation}/${dockerArchiver.DEFAULT_BUILD_LOCATION}/archive.tar`);
-
-  // TODO: provide something better?
-  return 'build complete';
+  return binaryBuild(config, `${config.projectLocation}/${dockerArchiver.DEFAULT_BUILD_LOCATION}/archive.tar`);
```

Comment:
i wonder if we need to return anything else other than the build info. Like pod name or something.  actually that is part of the deployment, so that would be in the applied-resource stuff, if we add it i guess